pkg/kube/controllers/quarkssecret/quarkssecret_reconciler.go

```diff
@@ -239,6 +250,70 @@ func (r *ReconcileQuarksSecret) createRSASecret(ctx context.Context, qsec *qsv1a
 	return r.createSecrets(ctx, qsec, secret)
 }
 
+// renderSecret uses the specified templating engine to draw the secret data
```

Comment:
doesn't render a secret, but a template

Reply:
it actually is rendering the `map[string]` which is directly consumed as secret data